### PR TITLE
fix: correct adding specific user as member

### LINKED ISSUE #224: Adding a second user to a role fails
Added two different literal users as editors fails. The first addition works; however, the second fails.

### PR DESCRIPTION
**Pull request by bjagg**
Hierarchy of forms was causing issues. Separated them to resolve issue.

Resolves #224

### CODE REVIEW

File: src/main/webapp/less/includes/_base.less [hunk 1]
```diff
@@ -178,4 +178,15 @@
     .no-col-padding {
         padding: 0;
     }
+
+    div.member-users {
+        margin-left: 2em;
+
+        div.member-user {
+
```
Comment (ChristianMurphy) [inline src/main/webapp/less/includes/_base.less:186]:
```suggestion
        div.member-user {
```

File: src/main/webapp/less/includes/_base.less [hunk 2]
```diff
@@ -178,4 +178,15 @@
     .no-col-padding {
         padding: 0;
     }
+
+    div.member-users {
```
Comment (ChristianMurphy) [inline src/main/webapp/less/includes/_base.less:182]: is div needed for specificity?
could it be removed?
```suggestion
    .member-users {
```

File: src/main/webapp/less/includes/_base.less [hunk 3]
```diff
@@ -178,4 +178,15 @@
...
+        margin-left: 2em;
+
+        div.member-user {
```
Comment (ChristianMurphy) [inline src/main/webapp/less/includes/_base.less:185]: is div needed for specificity?
could it be removed?
```suggestion
        .member-user {
```

**Review by ChristianMurphy — APPROVED** [review]
Thanks @bjagg!

**Review by jonathanmtran — APPROVED** [review]
I've deployed a snapshot build to a local uPortal-start instance and things appear to be working as they should. Thanks!

**Pull request merged**